scripts/ensure_owl_liquidity.js

```diff
@@ -65,6 +65,7 @@ module.exports = async (callback) => {
       SELL_ORDER_AMOUNT_OWL,
       artifacts
     )
+    // log.info(`Placed fee token liquidity orders for tokens: ${res}`)
```

Comment:
Remove?

Reply:
I plan to put this back in when I migrate this script to JS. I want to keep this text here. It will be removed/taken care of today.

Reply:
Took it out anyway.